client/my-sites/earn/ads/form-settings.jsx

```diff
@@ -213,6 +213,12 @@ class AdsFormSettings extends Component {
 		}, 0 );
 	}
 
+	jetpackPlacementControls() {
+		const linkHref = '/marketing/traffic/' + this.props.site.slug;
+
+		return <Card href={ linkHref }>Manage Ad Placements in Marketing Settings</Card>;
```

Comment:
Few thoughts here:
1) Title case seems out of place with other <click to do thing> links
2) We could probably shorten this to "Manage ad placements"
3) Should we set this to the self-hosted Jetpack Settings -> Traffic tab for Jetpack sites? When we finally update Calypso/Jetpack to be equivalent we can set it to the marketing/earn tab, but right now that tab doesn't do much for JP sites.

Reply:
We decided to push #3 in to a separate/TBD update.